packages/renderer/src/render.ts

```diff
@@ -30,6 +32,8 @@ export const renderFrames = async ({
 	imageFormat?: ImageFormat;
 	quality?: number;
 	browser?: Browser;
+	frameRange: FrameRange;
+	frames: number;
```

Comment:
kind of sucks but we might have to make it optional or release it as part of 2.0 breaking change. otherwise I think when you upgrade your project the types break